internal/api/config.go

```diff
@@ -1,5 +1,16 @@
 package api
 
+import "time"
+
 type Config struct {
-	Port string
+	Port             string
+	ShutdownDuration time.Duration
```

Comment:
```suggestion
	ShutdownTimeout time.Duration
```

WDYT?

Reply:
Timeout sounds better. Let me change it.